### ISSUE TITLE
[BUG] Weird oreDict Entry Names

### ISSUE THREAD

**Issue by Gliese-832-c** [ISurrealI/ContentCreator#9]
When creating materials with underscores, the automatic oreDict system converts them to spaces, as seen in the following screenshot:
![image](https://user-images.githubusercontent.com/55159077/179993474-dff04f94-367f-45f0-bcbe-34d92f207906.png)

This is the code used for the material:
```
val low_grade_lead = Material.get("low_grade_lead", 0x363849);

low_grade_lead.types(
	// Basic Components
	"ingot",
	"plate",
	"rod",
	"nugget",
	
	// Specialty Components
	"coin",
	"coinlarge"
);
```

**Comment by Gliese-832-c** [comment]
To elaborate on it - I changed it from low_grade_lead to lead_lowgrade for better internal organization issues. But not only does the name persist, now the localization is also messed up. I suggest being able to use custom localization overrides.
![image](https://user-images.githubusercontent.com/55159077/180002291-ec875d1c-8fec-49bc-89f5-7974aa1a6c72.png)

**Comment by Gliese-832-c** [comment]
I think I will make a separate issue for custom localization.

**Comment by ISurrealI** [comment]
This was an easy fix i think i'll expand #15
